core/services/relay/evm/cap_encoder.go

```
@@ -136,6 +137,15 @@ func (c *capEncoder) Encode(ctx context.Context, input values.Map) ([]byte, erro
 	return prependMetadataFields(meta, userPayload)
 }
 
+func (c *capEncoder) Decode(ctx context.Context, raw []byte, into any, _ string) error {
+	report, err := platform.Decode(raw)
+	if err != nil {
+		return err
+	}
+	c.codec.Decode(ctx, report.Data, into, encoderName)
```

Error is not returned